jest.config.js

```diff
@@ -155,7 +155,7 @@ const config = {
 
   // The glob patterns Jest uses to detect test files
   testMatch: [
-    '**/test/**/*.test.tsx', //'**/test/**/configmodel.test.tsx' 
+    '**/test/**/*.test.tsx', //'**/test/**/providermodal.test.tsx' 
```

Comment:
is the update by mistake?

Reply:
Actual code not changed. the changes is in commented line